code/modules/events/meteor_wave.dm

```diff
@@ -3,17 +3,20 @@
 /datum/round_event_control/meteor_wave
 	name = "Meteor Wave: Normal"
 	typepath = /datum/round_event/meteor_wave
-	weight = 4
+	weight = 7
```

Comment:
undocumented change

Reply:
okay i see why you do it but its still a balance change.

Reply:
I'd argue this is fixing a consistency issue. More intense versions of normal events are supposed to be less common, some epic prankster had inverted it for meteors.

Reply:
https://github.com/tgstation/tgstation/pull/32150 intended feature

Reply:
what a joker, that MSO character